src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -130,6 +130,7 @@ def load_arguments(self, _):
     enable_vtpm_type = CLIArgumentType(arg_type=get_three_state_flag(), min_api='2020-12-01', help='Enable vTPM.')
     enable_secure_boot_type = CLIArgumentType(arg_type=get_three_state_flag(), min_api='2020-12-01', help='Enable secure boot.')
     security_type = CLIArgumentType(arg_type=get_enum_type(['TrustedLaunch']), min_api='2020-12-01', help='Specify if the VM is Trusted Launch enabled. See https://docs.microsoft.com/azure/virtual-machines/trusted-launch.')
+    architecture_type = CLIArgumentType(arg_type=get_enum_type(self.get_models('Architecture', operation_group='disks')), min_api='2021-07-01', help='CPU architecture. Possible values include: x64, Arm64.')
```

Comment:
`arg_type=get_enum_type(self.get_models('Architecture', operation_group='disks')` and `min_api='2021-07-01'`  Are these two configurations the same in all operation groups?